lib/tooling/clang-tidy-tool.js

```diff
@@ -33,7 +33,10 @@ class ClangTidyTool extends BaseTool {
         super(toolInfo, env);
     }
 
-    runTool(sourcefile, args, compilerExe, outputFilename, options) {
+    runTool(compilationInfo, inputFilepath, args) {
+        const sourcefile = inputFilepath;
+        const compilerExe = "";
```

Comment:
Can you remove this (plus the check below on line 52) if it's not needed: seems like it's perhaps an old check for clang? Or is this functionality that needs to be reclaimed?

Reply:
ah no yeah this shouls refer to the compile exe, had something to do with clang compatible compilers that weren't named clang++ or something.